Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -631,6 +631,9 @@ public final class PackageBuilder {
 
         // Compute the path to public headers directory.
         let publicHeaderComponent = manifestTarget?.publicHeadersPath ?? ClangTarget.defaultPublicHeadersComponent
+        if publicHeaderComponent.first == "/" {
```

Comment:
We have a method for this: `RelativePath(validating:)`

Reply:
Ah, we should probably consolidate that with the non-validating initialiser, because that one only preconditions for / but not for ~